sdks/java/extensions/zetasketch/src/test/java/org/apache/beam/sdk/extensions/zetasketch/BigQueryHllSketchCompatibilityIT.java

```diff
@@ -65,23 +66,32 @@
   private static final List<String> TEST_DATA =
       Arrays.asList("Apple", "Orange", "Banana", "Orange");
 
-  // Data Table: used by testReadSketchFromBigQuery())
+  // Data Table: used by tests reading sketches from BigQuery
   // Schema: only one STRING field named "data".
-  // Content: prepopulated with 4 rows: "Apple", "Orange", "Banana", "Orange"
-  private static final String DATA_TABLE_ID = "hll_data";
   private static final String DATA_FIELD_NAME = "data";
   private static final String DATA_FIELD_TYPE = "STRING";
   private static final String QUERY_RESULT_FIELD_NAME = "sketch";
-  private static final Long EXPECTED_COUNT = 3L;
 
-  // Sketch Table: used by testWriteSketchToBigQuery()
+  // Content: prepopulated with 4 rows: "Apple", "Orange", "Banana", "Orange"
+  private static final String DATA_TABLE_ID_NON_EMPTY = "hll_data_non_empty";
+  private static final Long EXPECTED_COUNT_NON_EMPTY = 3L;
+
+  // Content: empty
+  private static final String DATA_TABLE_ID_EMPTY = "hll_data_empty";
```

Comment:
Does the aggregation (HLL_COUNT.INIT) over an empty table return a NULL sketch, as expected? I.e., did the test fail before you modified 'parseQueryResultToByteArray' to deal with NULLs? I think it should (according to https://plx.corp.google.com/scripts2/script_5d._6fe78c_0000_2144_8a38_883d24fc4a60, last SELECT), but double-checking. 
I'd wish we could add an assert, but that doesn't work with the utility method.

Reply:
Yes it does and I have tried it (although it is not mentioned in the BigQuery documentation).